docs/Configuration.md

```diff
@@ -714,12 +714,33 @@ url: the URL of the GBFS feed (do not include the gbfs.json at the end) *
 ```
 \* For a list of known GBFS feeds see the [list of known GBFS feeds](https://github.com/NABSA/gbfs/blob/master/systems.csv)
 
-#### TNC Configuration
+#### Transportation Network Company Configuration
 
-It is possible to add either Uber or Lyft as a TNC updater, assuming that either of those companies allow you to access their APIs. OTP provides helper methods for making authenticated requests to each provider's API and also for verifying that TNC service exists when making certain routing requests that use a TNC for part or all of the trip.
+A transportation network company (TNC) is a company that provides on-demand car hailing services. When one of these updaters is configured it is possible to plan trips where part or all of the journey involves a passenger being picked up, transported in a car and then dropped off at another location.
+
+A TNC updater can be configured with either a company that has an API for determining arrival and ride estimates, or with a simple "No API" updater that always returns a default arrival estimate and 0 duration and $0 ride estimates.
```

Comment:
Since this NOAPI mode is for testing, I think this line in the docs should clarify that.

Reply:
NOAPI isn't necessarily for testing. It could be run in production to simulate a TNC provider that didn't have an API to query.